src/main/java/com/griddynamics/cloud/learning/service/ArticleService.java

```diff
@@ -2,15 +2,98 @@
 
 import com.griddynamics.cloud.learning.dao.domain.Article;
 import com.griddynamics.cloud.learning.dao.domain.PaginatedResult;
+import com.griddynamics.cloud.learning.dao.repository.ArticleRepository;
 import com.griddynamics.cloud.learning.web.dto.ArticleFilterRequest;
-
-import java.util.Collections;
+import com.griddynamics.cloud.learning.web.dto.PaginatedRequestParams;
+import com.griddynamics.cloud.learning.web.dto.PurchaseType;
 
 public class ArticleService {
 
-    //TODO (Just a stub for now; part of service implementation task)
-    public PaginatedResult<Article> getArticles(ArticleFilterRequest requestFilter) {
+    private final ArticleRepository repository = new ArticleRepository();
+
+    /**
+     * Returns {@code PaginatedResult<Article>} dto from repository.
```

Comment:
Let's simplify it and make more obvious.

Example:
/**
     * Gets articles by given filter and sorting params.
     *
     * @param requestFilter - request dto, that contains sorting and filtering info.
     * @return - paginated list of articles.
     */

Reply:
Just use the provided example.